### ISSUE TITLE
Link tag in extent report not closed

### ISSUE THREAD

**Issue by dipesh17** [anshooarora/extentreports-java#1025]
## Summary
When Jenkins copying the report, parsing result file extentreport.html file to attach as an email to the team its throwing java.io.IOException: org.xml.sax.SAXParseException due to </link> is not closed in <head>

## Expected Behavior
Link tag should be closed.
**<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600' rel='stylesheet' type='text/css'>**

## Current Behavior
<link> tag is not closed which is causing issue.
`10:33:38 Unable to parse Report15_02_2018_10_24_32.html: java.io.IOException: org.xml.sax.SAXParseException; systemId: file:/var/lib/jenkins/jobs/Prelive-Selenium-Test/builds/35/seleniumReports/Report15_02_2018_10_24_32.html; lineNumber: 33; columnNumber: 4; The element type "link" must be terminated by the matching end-tag "</link>".`

## Sample

```java

// Link tag is not closed as a result when Jenkins prase the .html file it is throwing an error and unable to send as an attachment to the respective email address.

# Version 2.41.2
<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600' rel='stylesheet' type='text/css'>

# Version 3.1.3


<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600' rel='stylesheet' type='text/css'>

<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">


<link> Tag is not closed
```

## Environment Details

* Extent report Version used: 2.41.2 and 3.1.3
* Operating System and version: Linux or Mac
* JDK Version: 1.8

**Comment by dipesh17** [comment]
@anshooarora  Can you please help on the above, issue. it is stopping our CI process at the moment.

**Comment by anshooarora** [comment]
The tag can be made as self closing. Can you send in a pull request?

**Comment by dipesh17** [comment]
<img width="1316" alt="screen shot 2018-02-23 at 17 08 24" src="https://user-images.githubusercontent.com/26460424/36606641-41b6dc58-18bc-11e8-99d5-d3e03297b62c.png">

**Comment by anshooarora** [comment]
Can you send a pull request?

**Comment by dipesh17** [comment]
Can you please help me understand, why do you mean by sending you a pull request?

**Comment by anshooarora** [comment]
Can you only send the changes to affected files in the commit?  You may have to re-clone/re-fork the project to get the latest code before doing so.  If you are not able to, I will make the changes and close this.

**Comment by anshooarora** [comment]
Fixed in PR #1030
